lib/nexmo-jwt/jwt.rb

```diff
@@ -0,0 +1,31 @@
+require 'securerandom'
+require 'jwt'
+
+module Nexmo
+  class JWT
+    attr_accessor :application_id, :private_key, :jti, :nbf, :exp, :ttl, :acl_paths, :sub, :alg, :typ, :iat
+
+    def ttl
+      @ttl || 900
+    end
+
+    def generate
+      ::JWT.encode(to_payload, private_key, alg)
+    end
+
+    def to_payload
+      hash = {
+        iat: Time.now.to_i,
+        jti: jti || SecureRandom.uuid,
+        ttl: ttl,
```

Comment:
`ttl` doesn't need to be part of the hash, we're just using it to help generate `exp`

Reply:
@dragonmantank noted, and resolved!